src/app/components/Points/DelegationPoints.tsx

```diff
@@ -0,0 +1,31 @@
+import React from "react";
+
+import { useDelegationsPoints } from "@/app/context/api/DelegationsPointsProvider";
+
+interface DelegationPointsProps {
+  stakingTxHash: string;
+}
+
+export const DelegationPoints: React.FC<DelegationPointsProps> = ({
+  stakingTxHash,
+}) => {
+  const { delegationPoints, isLoading } = useDelegationsPoints();
+
+  const points = delegationPoints.get(stakingTxHash);
+
+  if (isLoading) {
+    return (
+      <div className="flex items-center justify-end gap-1">
+        <div className="h-5 w-12 animate-pulse rounded bg-gray-300 dark:bg-gray-700"></div>
+      </div>
+    );
+  }
+
+  return (
+    <div className="flex items-center justify-end gap-1">
+      <p className="whitespace-nowrap font-semibold">
+        {points !== undefined ? points : 0}
```

Comment:
Should we show plain 0? On mobile it might not be obvious, what this zero is: https://i.imgur.com/yu5ztXn.png

Reply:
should we show `-` instead? or some text representation?

Reply:
@vitsalis @jrwbabylonlab what do you think? I think on mobile we can do "Points: x", and on desktop just "X" since "Points" is the part of so-called "table"